docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -62,3 +63,70 @@ Log in the back office as an Administrator and go to the Users page. Select the
 
 Then log in again into the back office with your new user credentials.
 You have now the possibility to create new Rides and Points of Interest only.
+
+
+
+##  Customize the use forms
```

Comment:
`user registration forms`?

Also, why is this section in the step 6 file? It is not logically linked to the rest of the page, and more connected with step 7.

NB, some time in future it would make sense to move this to a point earlier in the tutorial, when the first view config is presented.

Reply:
:scream: yes big typo here!
The division is:
* in step 6, the user is just setting the pagelayout
* in step 7, it's the creation of new files to override the default ones

For the future, imho, if we move it earlier, keep in mind that our reader should be comfortable with the content in eZ Platform.

Reply:
Okay, everything into step 7: sold!